app/views/stories/destroy.js.erb

```diff
@@ -0,0 +1,4 @@
+function(){}
+  document.getElementById(`story_<%= @story.id %>`).remove()
+  console.log("The story was deleted.");
```

Comment:
Do we need this `console.log`? or it was just for debugging?

Reply:
A case for writing commits with additional description text 😋

Reply:
removed